content/applications/inventory_and_mrp/inventory/warehouses_storage/replenishment/resupply_warehouses.rst

```diff
@@ -1,70 +1,176 @@
+   from the central distribution center. Odoo :guilabel:`Inventory` manages these transfers to keep
+   stores in stock.
+
+This guide explains how to conduct an inter-warehouse transfer using replenishment strategies. There
+are two configuration methods for inter-warehouse replenishment:
+
+#. :ref:`Make to order (MTO) <inventory/warehouses_storage/MTO>`
+#. :ref:`Reordering rule <inventory/warehouses_storage/reordering-rule>`
+
+.. seealso::
+   :doc:`Difference between MTO and reordering rules <../replenishment>`
 
 Configuration
 =============
 
-To resupply from another warehouse, first go to :menuselection:`Inventory --> Configuration -->
-Settings --> Warehouse` and activate :guilabel:`Multi-Step Routes`. Then, click :guilabel:`Save` to
-apply the setting.
+To resupply from another warehouse, first go to :menuselection:`Inventory app --> Configuration -->
+Settings`. In the :guilabel:`Warehouse` section, activate :guilabel:`Storage Locations`. Then, click
+:guilabel:`Save` to apply the setting.
 
-.. image:: resupply_warehouses/virtual-warehouses-settings.png
+.. image:: resupply_warehouses/storage-locations.png
    :align: center
-   :alt: Enable Multi-Step Routes in Inventory settings.
+   :alt: Enable Storage Locations in Inventory settings.
 
-View all the configured warehouses by going to :menuselection:`Inventory --> Configuration -->
-Warehouses`.
+Warehouses
+----------
 
-Create a new warehouse by clicking :guilabel:`Create`. Then, give the warehouse a name and a
-:guilabel:`Short Name`. Finally, click :guilabel:`Save` to finish creating the warehouse.
+Configure the settings for the central warehouse and retail shops by going to
+:menuselection:`Inventory app --> Configuration --> Warehouses`.
 
-After that, go back to the :guilabel:`Warehouses` page and open the warehouse that will be
-resupplied by the second warehouse. Then, click :guilabel:`Edit`. In the :guilabel:`Warehouse
-Configuration` tab, locate the :guilabel:`Resupply From` field, and check the box next to the
-second warehouse's name. If the warehouse can be resupplied by more than one warehouse, make sure
-to check those warehouses' boxes too. Finally, click :guilabel:`Save` to apply the setting. Now,
-Odoo knows which warehouses can resupply this warehouse.
+.. important::
+   Each central warehouse and retail shop *must* have its own warehouse. In other words, each shop
+   is considered a local warehouse.
 
-.. image:: resupply_warehouses/resupply-from-second-warehouse.png
+Create a new warehouse by clicking :guilabel:`New`. Then, give the warehouse a name and a
+:guilabel:`Short Name`, which will appear on that warehouse's transfers. Finally, click
+:guilabel:`Save` to finish creating the warehouse.
+
+After that, go back to the :guilabel:`Warehouses`, page and open the warehouse that will be
+resupplied by the central warehouse. In the :guilabel:`Warehouse Configuration` tab, locate the
```

Comment:
Framing is a but strange?
Why would we create a warehouse, then go back to the dashboard to possibly click into the form again?

This can be framed more clearly and without that possibility of interpretation like:
"To create a new warehouse do this...OR pick the existing warehouse..."